src/sentry/static/sentry/app/views/organizationIncidents/detail/index.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import LoadingIndicator from 'app/components/loadingIndicator';
+import LoadingError from 'app/components/loadingError';
+import {PageContent} from 'app/styles/organization';
+import withApi from 'app/utils/withApi';
+
+import IncidentHeader from './header';
+import IncidentDetailsBody from './body';
+
+class IncidentDetails extends React.Component {
```

Comment:
Should this be an `AsyncView`?

Reply:
I had this initially, but since this is going to be a very heavily customized view, I think it's likely going to be more work to actually try to make AsyncView fit what we want to do achieve here in the UI.